src/ContentHub.php

```diff
@@ -12,15 +12,24 @@
 
 class ContentHub extends Client
 {
+
+    /**
+     * Defines the Content Hub API Version.
+     *
+     * @var string
+     */
+    private $api_version;
+
     /**
      * Overrides \GuzzleHttp\Client::__construct()
      *
      * @param string $apiKey
      * @param string $secretKey
      * @param string $origin
      * @param array  $config
+     * @param string $api_version
      */
-    public function __construct($apiKey, $secretKey, $origin, array $config = [])
+    public function __construct($apiKey, $secretKey, $origin, array $config = [], $api_version = NULL)
     {
```

Comment:
style question: any reason to have $api_version be NULL... instead of type declaring as a string and defaulting it to'v1'. Then you could blindly assign on line 45 without the !empty check.